proposals/digital-credential-proposal.md

```diff
@@ -1,4 +1,10 @@
-This is an alternative design that optimizes for decoupling federation and wallets further (so that they have more autonomy to evolve independently) 
+# ⚠️ This proposals has been superseded ⚠️
+
+- Please refer to the [Digital identities spec](https://wicg.github.io/digital-identities/) instead. It represents the community's consensus.
+
+## Alternative proposal
+
+This is an alternative design that optimizes for decoupling federation and wallets further (so that they have more autonomy to evolve independently)
 while still accomplishing the goal of being able to request them at the same time in the same dialog.
```

Comment:
```suggestion
This is an alternative design that further optimizes for decoupling federation and wallets (so that they have more autonomy to evolve independently),
while still accomplishing the goal of being able to request both at the same time in a single dialog.
```